psi4/src/psi4/libmints/eribase.cc

```diff
@@ -2890,26 +2909,16 @@ size_t TwoElectronInt::compute_quartet_deriv1(int sh1, int sh2, int sh3, int sh4
                             libderiv_.PrimQuartet[nprim].U[4][i] = WP[i];
                             libderiv_.PrimQuartet[nprim].U[5][i] = WQ[i];
                         }
-                        libderiv_.PrimQuartet[nprim].oo2z = oo2z;
-                        libderiv_.PrimQuartet[nprim].oo2n = oo2n;
-                        libderiv_.PrimQuartet[nprim].oo2zn = oo2zn;
-                        libderiv_.PrimQuartet[nprim].poz = rho * ooz;
-                        libderiv_.PrimQuartet[nprim].pon = rho * oon;
-                        // libderiv_.PrimQuartet[nprim].oo2p = oo2rho;   // NOT SET IN CINTS
-                        libderiv_.PrimQuartet[nprim].twozeta_a = 2.0 * a1;
-                        libderiv_.PrimQuartet[nprim].twozeta_b = 2.0 * a2;
-                        libderiv_.PrimQuartet[nprim].twozeta_c = 2.0 * a3;
-                        libderiv_.PrimQuartet[nprim].twozeta_d = 2.0 * a4;
 
                         double T = rho * PQ2;
+                        fjt_->set_rho(rho);
```

Comment:
Good catch finding this missing! Glad you changed the hessian code, too.

Reply:
This was a hard earned LoC.  Preferences for Speyside, Islay, Highland?